packages/ember-engines/addon/components/link-to-component.js

```diff
@@ -1,13 +1,21 @@
-import LinkComponent from '@ember/routing/link-component';
-import { getOwner } from '@ember/application';
+import RoutingLinkComponent from '@ember/routing/link-component';
```

Comment:
@BobrImperator I may be wrong, but this line would cause build time error with Ember v4. I assume we need to do

```js
LinkComponent = importSync('@ember/routing/link-component');
```

one line 15

Reply:
This won't work because `@ember` dependencies are transpiled to use global `Ember` in older versions.

Actually the hack is to do `import { LinkTo as RoutingLinkComponent } from '@ember/routing';`
just like in `link-to-external` but for some reason here it's not failing.

This PR is already passing Ember 4